src/components/new-request/search.vue

```diff
@@ -348,12 +366,38 @@ export default class NewSearch extends Mixins(CommonMixin) {
     })
   }
 
-  // called when switching between request and manage tabs (which are cached)
+  /** Called when switching between request and manage tabs (which are cached). */
   private activated () {
     this.scrollTo('namerequest-sbc-header')
   }
 
-  // Local Getters
+  get isNewBcBusiness (): boolean {
+    return (this.request?.value === NrRequestActionCodes.NEW_BUSINESS && this.request?.group === 0)
+  }
+
+  get isContinuationIn (): boolean {
+    return (this.request?.value === NrRequestActionCodes.MOVE)
+  }
+
+  get isAlterType (): boolean {
+    return (this.request?.value === NrRequestActionCodes.CONVERSION)
+  }
+
+  get showJurisdiction (): boolean {
+    if (!this.request) return false
+    if (this.isNewBcBusiness) return false
+    if (this.isContinuationIn) return false
+    if (this.isAlterType) return false
+    return true
+  }
```

Comment:
Simple rules -- should be easy to build upon.

Reply:
These'll help with future tickets (workflows), thanks!